docs/BR.md

```diff
@@ -1739,142 +1739,875 @@ The CA SHALL enforce multi-factor authentication for all accounts capable of dir
+| `extKeyUsage`                     | MUST         | -            | See [Section 7.1.2.7.4](#71274-extended-key-usage) |
+| `id-pkix-ocsp-nocheck`            | MUST         | N            | See [Section 7.1.2.7.5](#71275-id-pkix-ocsp-nocheck) |
+| `keyUsage`                        | MUST         | Y            | See [Section 7.1.2.7.6](#71276-key-usage) |
+| `certificatePolicies`             | MUST NOT     | N            | - |
+| `nameConstraints`                 | MUST NOT     | -            | - |
+| `subjectAltName`                  | MUST NOT     | -            | - |
+| `subjectKeyIdentifier`            | SHOULD       | N            | See [Section 7.1.2.9.5](#71295-subject-key-identifier) |
+| `authorityInformationAccess`      | SHOULD NOT   | N            | See [Section 7.1.2.7.2](#71272-authority-information-access) |
+| `crlDistributionPoints`           | SHOULD NOT   | N            | See [Section 7.1.2.9.3](#71293-crl-distribution-points) |
+| Signed Certificate Timestamp List | MAY          | N            | See [Section 7.1.2.9.4](#71294-signed-certificate-timestamp-list) |
+| Any other extension               | SHOULD NOT   | -            | __**TBD**__ |
+
+##### 7.1.2.7.2 Authority Information Access
+
+For OCSP Responder certificates, this extension SHOULD NOT be present, as the Relying Party should already possess the necessary information. In order to validate the given Responder certificate, the Relying Party must have access to the Issuing CA's certificate, eliminating the need to provide `id-ad-caIssuers`. Similarly, because of the requirement for an OCSP Responder certificate to include the `id-pkix-ocsp-nocheck` extension, it is not necessary to provide `id-ad-ocsp`, as such responses will not be checked by Relying Parties.
+
+If present, the `AuthorityInformationAccesssSyntax` MUST contain one or more `AccessDescription`s. Each `AccessDescription` MUST only contain a permitted `accessMethod`, as detailed below, and each `AuthorityInformationAccessSyntax` MUST contain all required `AccessDescription`s.
+
+| __Access Method__ | __OID__            | __Presence__ | __Access Location__         | __Description__ |
+| --                | --                 | -            | ----                        | ---             |
+| `id-ad-caIssuers` | 1.3.6.1.5.5.7.48.2 | SHOULD NOT   | `uniformResourceIdentifier` | An HTTP URL of the Issuing CA's certificate. |
+| `id-ad-ocsp`      | 1.3.6.1.5.5.7.48.1 | MUST NOT     | `uniformResourceIdentifier` | An HTTP URL of the Issuing CA's OCSP responder. |
+| Any other value   | -                  | MUST NOT     | -                           | -               |
+
+##### 7.1.2.7.3 Basic Constraints
+
+| __Field__           | __Description__ |
+| ---                 | ------- |
+| `cA`                | MUST be FALSE |
+| `pathLenConstraint` | MUST NOT be present |
+
+**Note**: CAs MUST observe DER encoding rules, such as not explicitly encoding DEFAULT values within OPTIONAL fields.
```

Comment:
I mean, I'd rather remove this language entirely, because a CA should be familiar with X.690 in total because it's a normative dependency of RFC 5280.

Reply:
I'm not sure I fully agree with the approach of "every requirement is in the normative references of normative references, good luck", because in the past, especially when the SCWG sees repeated failures or bad practices from CAs, there is an effort to guide CAs to follow good practices or to avoid some mistakes. Obviously this doesn't dismiss CA's responsibility for following all normative references and complying with everything but we all acknowledge the difficulties and often subtleties of interpreting some requirements.

I believe the BRs have a good balance of not duplicating everything from other normative documents but highlighting areas of confusion or "easy-to-miss" requirements that are referenced 1-2 levels away from the BRs. 

This particular Note comes from an old Mozilla guidance post because several CAs had missed it. Adding something useful for the benefit of the ecosystem doesn't seem to cause harm IMHO.

Reply:
Split the difference and reference RFC 5280 Appendix B?

> Implementers should note that the DER encoding of SET or SEQUENCE
   components whose value is the DEFAULT omit the component from the
   encoded certificate or CRL.  For example, a BasicConstraints
   extension whose cA value is FALSE would omit the cA boolean from the
   encoded certificate.

Reply:
Shifted this to 7.1.2.5 and removed this text. Let me know if this doesn't solve the issue by reopening this :)